pkg/views/server/configure.go

```diff
@@ -36,6 +42,32 @@ func ConfigurationForm(config *serverapiclient.ServerConfig) *serverapiclient.Se
 			huh.NewInput().
 				Title("Server Download URL").
 				Value(config.ServerDownloadUrl),
+			huh.NewInput().
+				Title("API Port").
+				Value(&apiPortView).
+				Validate(createPortValidator(config, &apiPortView, config.ApiPort)),
+			huh.NewInput().
+				Title("Headscale Port").
+				Value(&headscalePortView).
+				Validate(createPortValidator(config, &headscalePortView, config.HeadscalePort)),
+			huh.NewInput().
+				Title("Binaries Path").
+				Value(config.BinariesPath),
```

Comment:
@Twacqwq we should have validation for "Binaries Path" that the input values is a directory that exists and for "Targets File Path" that it's a file that exists.

Reply:
sure. I will update it.

Reply:
@Twacqwq please make sure to rebase with main again